index.html

```diff
@@ -0,0 +1,18 @@
+<h1>Jachin Antalan</h1>
+<h2>About</h2>
+<p>
+  Code The Dream Student - Haumea Class I am a pet lover. My hobbies include
+  playing board games and video games.
+</p>
+<h2>Experience</h2>
+<ul>
+  <li>Graphic Design</li>
+  <li>Digital Marketing</li>
+</ul>
+<h2>Connect</h2>
+<ul>
+  <li><a href="https://github.com/chinsomnia">Github</a></li>
+  <li><a href="https://www.linkedin.com/in/chinantalan/">LinkedIn</a></li>
+  <li><a href="https://www.instagram.com/chinstagrammed">Instagram</a></li>
+  <li><a href="jachinantalan@gmail.com">Email</a></li>
```

Comment:
For emails, the href is usually written as `href="mailto:jachinantalan@gmail.com"`.

https://www.w3docs.com/snippets/html/how-to-create-mailto-links.html